packages/cli/test/fixtures/repository/index.js

```diff
@@ -0,0 +1,161 @@
+const DATASOURCE_APP_PATH = 'src/datasources';
+const MODEL_APP_PATH = 'src/models';
+const CONFIG_PATH = '.';
+const DUMMY_CONTENT = '--DUMMY VALUE--';
+
+exports.SANDBOX_FILES = [
+  {
+    path: CONFIG_PATH,
+    file: 'myconfig.json',
+    content: `{
+      "datasource": "dbmem",
+      "model": "decoratordefined"
+    }`,
+  },
+  {
+    path: DATASOURCE_APP_PATH,
+    file: 'dbkv.datasource.json',
+    content: JSON.stringify({
+      name: 'dbkv',
+      connector: 'kv-redis',
+    }),
+  },
+  {
+    path: DATASOURCE_APP_PATH,
+    file: 'dbkv.datasource.ts',
+    content: DUMMY_CONTENT,
+  },
+  {
+    path: DATASOURCE_APP_PATH,
+    file: 'dbmem.datasource.json',
+    content: JSON.stringify({
+      name: 'dbmem',
+      connector: 'memory',
+    }),
+  },
+  {
+    path: DATASOURCE_APP_PATH,
+    file: 'dbmem.datasource.ts',
+    content: DUMMY_CONTENT,
+  },
+  {
+    path: DATASOURCE_APP_PATH,
+    file: 'restdb.datasource.json',
+    content: JSON.stringify({
+      name: 'restdb',
+      connector: 'rest',
+    }),
+  },
+  {
+    path: DATASOURCE_APP_PATH,
+    file: 'restdb.datasource.ts',
+    content: DUMMY_CONTENT,
+  },
+  {
+    path: MODEL_APP_PATH,
+    file: 'decoratordefined.model.ts',
+    content: `
+    import {Entity, model} from '@loopback/repository';
+
+    @model({
+      name: 'Product',
+      properties: {
+        thePK: {type: 'number', id: true},
+        name: {type: 'string', required: true},
+      },
+    })
+    export class DecoratorDefined extends Entity {
+      thePK: number;
+      name: string;
+
+      constructor(data?: Partial<DecoratorDefined>) {
+        super(data);
+      }
+    }
+    `,
+  },
+  {
+    path: MODEL_APP_PATH,
+    file: 'defaultmodel.model.ts',
+    content: `
+    import {Entity, model, property} from '@loopback/repository';
+
+    @model()
+    export class DefaultModel extends Entity {
+      @property({
+        type: 'number',
+        id: true,
+        default: 0,
+      })
+      id?: number;
+
+      @property({
+        type: 'string',
+      })
+      desc?: string;
+
+      @property({
+        type: 'number',
+        default: 0,
+      })
+      balance?: number;
+
+      constructor(data?: Partial<DefaultModel>) {
+        super(data);
+      }
+    }
+    `,
+  },
+  {
+    path: MODEL_APP_PATH,
+    file: 'multi-word.model.ts',
+    content: `
+    import {Entity, model, property} from '@loopback/repository';
+
+    @model()
+    export class MultiWord extends Entity {
+      @property({
+        type: 'string',
+        id: true,
+        default: 0,
+      })
+      pk?: string;
+
+      @property({
+        type: 'string',
+      })
+      desc?: string;
+
+      constructor(data?: Partial<MultiWord>) {
+        super(data);
+      }
+    }
+    `,
+  },
+  {
+    path: MODEL_APP_PATH,
+    file: 'invalid-id.model.ts',
+    content: `
```

Comment:
Maintaining source code in a string template literal is cumbersome. Can you move the source code into a standalone file please, e.g. `packages/cli/test/fixtures/repository/models/invalid-id.model.ts`.

Then you can e.g. read the file here:

```ts
content: fs.readFileSync(require.resolve('./models/invalid-id.model.ts'), {encoding: 'utf-8'}),
```

It would be even better if the code creating sandbox directory was able to copy a tree of files from fixtures directory, so that these instructions in `SANDBOX_FILES` can say something along the lines of "recursively copy content of the directory `packages/cli/test/fixtures/repository`".